docs/tutorials/advanced/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How to create a new Expectation Suite using Rule Based Profilers
+---
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+<Prerequisites>
+
+- Have configured a Datasource to connect to your data
```

Comment:
I'm a bit torn on this. The article shows how to configure a datasource, therefore this line could perhaps be removed. And the tutorial also points to another guide to have the user set up a datasource. Let me think on this a bit more.

Reply:
I think by nature of this being a more complex or advanced guide, explicitly stating the prerequisites (even if we briefly touch upon them again) is of value.

Knoxpod definitely has a better grasp on this so I'll trust their opinion but I'd love to see what the outcome here is!